### DISCUSSION TITLE
Triggerer: gitlab-python

### DISCUSSION THREAD

**Discussion by jalanh11640** [python-gitlab/python-gitlab#2723] [Q&A] [answered]
Our project has a script that we use to push our changes to the remote server.  This of course, kicks off a pipeline.  When I look at the pipeline in the browser, the 'Triggerer' column shows 'gitlab-python'.  Is there a setting in our script I can set that will make the triggerer the user who's running the script?

Thanks!

**Comment by nejch** [comment, marked as answer]
Hi @jalanh11640. The `Triggerer` in the GitLab UI has nothing to do with the python-gitlab itself, but the the owner of the token  you provide.

So I guess you use a bot/project token that has that name. Simply rename that, or if you trigger the script from a pipeline, check if it's possible to authenticate using `CI_JOB_TOKEN` instead of an access token - since the job token always takes the identity of the user starting the job.

**Reply by jalanh11640** [reply]
Thank you for the response. That totally make sense.
